SECURITY.md

```diff
@@ -0,0 +1,106 @@
+# Security Release Process
+
+The community has adopted this security disclosure and response policy to ensure we responsibly handle critical issues.
+
+
+## Supported Versions
+
+For a list of support versions that this project will potentially create security fixes for, please refer to the Releases page on this project's GitHub and/or project related documentation on release cadence and support.
+
+
+## Reporting a Vulnerability - Private Disclosure Process
+
+Security is of the highest importance and all security vulnerabilities or suspected security vulnerabilities should be reported to this project privately, to minimize attacks against current users  before they are fixed. Vulnerabilities will be investigated and patched on the next patch (or minor) release as soon as possible. This information could be kept entirely internal to the project.
+
+If you know of a publicly disclosed security vulnerability for this project, please **IMMEDIATELY** contact the maintainers of this project privately. The use of encrypted email is encouraged.
+
+
+**IMPORTANT: Do not file public issues on GitHub for security vulnerabilities**
+
+To report a vulnerability or a security-related issue, please contact the VMware email address with the details of the vulnerability. The email will be fielded by the maintainers who have committer and release permissions. Emails will be addressed within 3 business days, including a detailed plan to investigate the issue and any potential workarounds to perform in the meantime. Do not report non-security-impacting bugs through this channel. Use GitHub issues instead.
```

Comment:
> please contact the VMware email address

Do we have a default email address to use for this? This seems strangely worded to me. Maybe it should be, please contact a maintainer with a VMware email address? Do we require maintainers to include their VMware email address on their GH profiles? If not, maybe we should for this reason?

Reply:
We do not have an email address that can be used for all projects. The security@vmware.com can only be used for our higher tier projects.

Reply:
Also, emails to the maintainers may or may not be public based on their profile. Maybe we should consider that.